test/push_sender_test.js

```diff
@@ -27,11 +44,21 @@ describe('push notification sender', function() {
   it('should emit push notifications on new messages', function*() {
     this.timeout(10000);
 
+    const family = 'family_name';
+    const id = 1;
     const message = {
-      reminder: 'Random data',
-      subscription: {
+      reminder: {
         id: 1,
```

Comment:
ok no, I don't get it :) you need to name it better because I think this is the reminderId but you used it in the subscription object.